lib/restricted_http/private_network_guard.rb

```diff
@@ -16,7 +16,7 @@ def resolve(hostname)
 
     def private_ip?(ip)
       IPAddr.new(ip).then do |ipaddr|
-        ipaddr.private? || ipaddr.loopback? || LOCAL_IP.include?(ipaddr)
+        ipaddr.private? || ipaddr.loopback? || ipaddr.link_local? || LOCAL_IP.include?(ipaddr)
```

Comment:
@jeremy we patched up a few more exploits in Fizzy's webhooks like publishing to broadcasts and using ipv4 mapped to ipv6.
Would you be OK with me adding that to your PR?

Reply:
Sure @monorkin